napari/utils/_appdirs.py

```diff
@@ -21,7 +21,7 @@
     appdirs.user_data_dir, _appname, _appauthor
 )
 user_config_dir: Callable[[], str] = partial(
-    appdirs.user_config_dir, _appname, _appauthor
+    appdirs.user_config_dir, _appname, _appauthor, version_string
```

Comment:
Not for this PR, but I wonder if we should consider making all these directories versioned? I'm not sure what some of them are (e.g. `user_state_dir`) though. If so, we might want to create an issue to track that work.

Reply:
As far as my searches suggest, `user_state_dir`, `user_log_dir`, and `user_data_dir` are not used anywhere in the napari codebase.
But they are not private, so dunno if someone is using them?